<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextFileExplorer</class>
 <widget class="QMainWindow" name="TextFileExplorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QWidget" name="searchLayout" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="searchTextLabel">
         <property name="text">
          <string>Szukaj tekstu:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchTextEdit"/>
       </item>
       <item>
        <widget class="QCheckBox" name="caseCheckBox">
         <property name="text">
          <string>Uczulony</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="wholeCheckBox">
         <property name="text">
          <string>Cale slowa</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="fileButtonsLayout" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="openButton">
         <property name="text">
          <string>Otworz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>Zamknij</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitButton">
         <property name="text">
          <string>Wyjście</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QWidget" name="searchButtonsLayout" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="previousButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Poprzedni</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Nastepny</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Wyczyść</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionOtworz"/>
    <addaction name="actionZamknij"/>
    <addaction name="actionWyjscie"/>
   </widget>
   <widget class="QMenu" name="menuSzukaj">
    <property name="title">
     <string>Szukaj</string>
    </property>
    <addaction name="actionPoprzedni"/>
    <addaction name="actionNastepny"/>
    <addaction name="actionWyczysc"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuSzukaj"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOtworz">
   <property name="text">
    <string>Otworz</string>
   </property>
  </action>
  <action name="actionZamknij">
   <property name="text">
    <string>Zamknij</string>
   </property>
  </action>
  <action name="actionWyjscie">
   <property name="text">
    <string>Wyjscie</string>
   </property>
  </action>
  <action name="actionPoprzedni">
   <property name="text">
    <string>Poprzedni</string>
   </property>
  </action>
  <action name="actionNastepny">
   <property name="text">
    <string>Nastepny</string>
   </property>
  </action>
  <action name="actionWyczysc">
   <property name="text">
    <string>Wyczysc</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionOtworz</sender>
   <signal>triggered()</signal>
   <receiver>TextFileExplorer</receiver>
   <slot>onOpenButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNastepny</sender>
   <signal>triggered()</signal>
   <receiver>TextFileExplorer</receiver>
   <slot>onNextButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPoprzedni</sender>
   <signal>triggered()</signal>
   <receiver>TextFileExplorer</receiver>
   <slot>onPreviousButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWyczysc</sender>
   <signal>triggered()</signal>
   <receiver>TextFileExplorer</receiver>
   <slot>onClearButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWyjscie</sender>
   <signal>triggered()</signal>
   <receiver>TextFileExplorer</receiver>
   <slot>onExitButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZamknij</sender>
   <signal>triggered()</signal>
   <receiver>TextFileExplorer</receiver>
   <slot>onCloseButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>TextFileExplorer</receiver>
   <slot>onOpenButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>630</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>473</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>TextFileExplorer</receiver>
   <slot>onCloseButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>633</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>TextFileExplorer</receiver>
   <slot>onExitButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>621</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>662</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>previousButton</sender>
   <signal>clicked()</signal>
   <receiver>TextFileExplorer</receiver>
   <slot>onPreviousButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextButton</sender>
   <signal>clicked()</signal>
   <receiver>TextFileExplorer</receiver>
   <slot>onNextButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>TextFileExplorer</receiver>
   <slot>onClearButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onOpenButtonClicked()</slot>
  <slot>onCloseButtonClicked()</slot>
  <slot>onExitButtonClicked()</slot>
  <slot>onPreviousButtonClicked()</slot>
  <slot>onNextButtonClicked()</slot>
  <slot>onClearButtonClicked()</slot>
 </slots>
</ui>
